include/swift/AST/Evaluator.h

```diff
@@ -452,6 +452,6 @@ evaluateOrFatal(Evaluator &eval, Request req) {
   });
 }
 
-} // end namespace evaluator
+} // end namespace swift
```

Comment:
Accidental change? 🙂

Reply:
Nope, this is intentional. The comment was wrong, there is no such namespace in this header.